include/worker.hpp

```diff
@@ -100,6 +100,39 @@ class Worker
                       types::ObjectMap& objectInterfaceMap,
                       const std::string& vpdFilePath);
 
+    /**
+     * @brief An API to Restore and Backup VPD.
+     *
+     * Note: This API works on set of pre-declared keywords to,
+     *
+     * Restore primary keyword value with secondary keyword value,
+     * when primary keyword has default value but
+     * secondary has non default value.
+     *
+     * Backup the primary keyword value on secondary,
+     * when primary keyword has non default value but
+     * secondary has default value.
+     *
+     * @param[in,out] io_parsedVpdMap - Parsed VPD map.
+     */
+    void backupAndRestore(types::VPDMapVariant& io_parsedVpdMap);
+
+    /**
+     * @brief An API to get primary EEPROM file path.
+     *
+     * @return path, primary EEPROM file path if present in the
+     * config JSON, empty otherwise.
+     */
+    std::string getPrimaryHwPath();
```

Comment:
Why this is a public API?

Reply:
getSecondaryHwPath() is being used in Manager, so kept both getPrimaryHwPath and getSecondaryHwPath API as public.